src/app/views/collections/CollectionsController.test.js

```diff
@@ -311,4 +321,15 @@ describe("mapStateToProps function", () => {
     it("returns the application's root path", () => {
         expect(mappedProps.rootPath).toBe("/florence");
     });
+});
+
+describe("Viewer user types are able to see all collections, even completed", () => {
```

Comment:
Possibly consider adding an `expect` statement on on the collection state as well - just to be sure.

Also I'm not sure where the mocked collections come from (so dismiss this if it's nonsense) but in the mocked collections list used by these tests is there a mix of collections in various states _i.e._ 3 collections with 2 in progress 1 completed?
I'm sure it's fine but I think it might be worth while adding a few different cases to verify this does work as expected.

Reply:
Thanks for the review, yep definitely worth having multiple states of collections to test this on, in this case, there were a set of mock collections available made up of 5 collections each with different states/properties. I am pushing some improvements now.